### PR TITLE
[3117] Show provider name in record card and record when signed in as an admin

### PR DESCRIPTION
**Pull request by defong**
### Context
Showing provider name / provider name and code

### Changes proposed in this pull request
Showing provider name for application record card
Showing provider name and code for record details

### Guidance to review
Any of the *`_as_system_admin`
https://register-pr-1705.london.cloudapps.digital/view_components/application_record_card/view

https://register-pr-1705.london.cloudapps.digital/view_components/record_details/view

image

image

### CODE REVIEW

**Comment by defong** [comment]
[Show provider name in record card and record when signed in as an admin](https://trello.com/c/6t3rpAkc/3117-show-provider-name-in-record-card-and-record-when-signed-in-as-an-admin)

File: app/components/application_record_card/view.rb [hunk 1]
```diff
@@ -48,6 +49,12 @@ def trainee_id
       tag.p("Trainee ID: #{record.trainee_id}", class: "govuk-caption-m govuk-!-font-size-16 application-record-card__id govuk-!-margin-top-0 govuk-!-margin-bottom-1")
     end
 
+    def provider_name
+      return unless system_admin
+
+      tag.p("Provider: #{record.provider.name}", class: "govuk-caption-m govuk-!-font-size-16 application-record-card__provider_name govuk-!-margin-top-0 govuk-!-margin-bottom-1")
```
Comment (archferns) [inline app/components/application_record_card/view.rb:55]:
```suggestion
      tag.p(record.provider.name, class: "govuk-caption-m govuk-!-font-size-16 application-record-card__provider_name govuk-!-margin-top-0 govuk-!-margin-bottom-1")
```
I think we don't need the "Provider:" bit?

Comment (defong) [reply app/components/application_record_card/view.rb:55]: image

Its that bit...

**Review by archferns — APPROVED** [review]

File: app/components/application_record_card/view.rb (hunk 1 above)
Comment (defong) [reply app/components/application_record_card/view.rb]: Actually you was right :heart:

**Review by edwardhorsford — APPROVED** [review]

**Pull request merged**